js/gallery/src/Gallery.svelte

```diff
@@ -229,15 +222,14 @@
 	>
 		<div
 			class="grid-container"
-			style="{grid_cols_style} {grid_rows_style}"
-			style:object_fit
+			style="{grid_cols_style} {grid_rows_style} --object-fit: {object_fit}; height: {height}"
```

Comment:
why is height set here? it's already set in the parent Block in js/app/src/components/Gallery/Gallery.svelte

Reply:
Need here otherwise the parameter is not respected in the actual gallery size:

![need_height_css](https://github.com/gradio-app/gradio/assets/41651716/085d6a8e-19c6-4350-bcaa-7927cdc3e663)